src/main/java/com/unstoppabledomains/util/DnsRecord.java

```diff
@@ -0,0 +1,21 @@
+package com.unstoppabledomains.util;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.ToString;
+
+@Getter
+@AllArgsConstructor
+@ToString
+public class DnsRecord {
+  private DnsRecordsType type;
+  private Number ttl;
```

Comment:
It's a bad idea to use `Number` since it's an abstract type. Please, replace it with something concrete like `Integer`, `Long`, `BigInteger`, `Double`, etc.